packages/xod-client-electron/src/app/arduinoActions.js

```diff
@@ -333,6 +370,7 @@ export const uploadToArduinoHandler = (event, payload) => {
         error: err,
         message,
         percentage,
+        additionalData,
```

Comment:
Can’t get the idea why we have to carry the IDs map through everything in the `arduinoActions.js` again and again. Sometimes with such implicit stuff?